vochain/state/processblockregistry.go

```diff
@@ -0,0 +1,83 @@
+package state
+
+import (
+	"encoding/binary"
+
+	"go.vocdoni.io/dvote/statedb"
+)
+
+type ProcessBlockRegistry struct {
+	startBlocksDB statedb.Updater
+	dbLock        *treeTxWithMutex
+	mainTree      statedb.TreeViewer
+}
+
+// RegisterStartBlock method creates a new record on the key-value database
```

Comment:
Just a quick remember to keep the comments starting whit the function o method name